plugins/CoreUpdater/Diagnostic/HttpsUpdateCheck.php

```diff
@@ -30,19 +30,24 @@ public function __construct(Translator $translator)
 
     public function execute()
     {
+        $faqLinks = [
+          '<a href="https://matomo.org/faq/faq-how-to-disable-https-for-matomo-org-and-api-matomo-org-requests" rel="noreferrer noopener" target="_blank">',
+          '</a>'
+        ];
         $label = $this->translator->translate('Installation_SystemCheckUpdateHttps');
 
-        if (GeneralConfig::getConfigValue('force_matomo_ssl_request') === 0) {
-            //if config is off, show info
-            $comment = $this->translator->translate('Installation_MatomoSslRequestConfigInfo');;
+        if (GeneralConfig::getConfigValue('force_matomo_http_request') === 1) {
```

Comment:
I guess @sgiehl already commented that you can't do strict comparisons on Config settings as they could be strings. Can you please fix this one and all others?